app/schemas/selectors/cinematic.js

```diff
@@ -321,3 +339,9 @@ export const getBackgroundObject = compose(triggers, backgroundObject)
  * @returns {number|undefined} the delay before removing the background object.
  */
 export const getClearBackgroundObject = compose(triggers, clearBackgroundObject)
+
+/**
+ * @param {Shot} shot
+ * @returns {Object|undefined}
+ */
+export const getCamera = compose(shotSetup, camera)
```

Comment:
Having trouble telling the difference between the `camera` selector and this selector.  From what I can tell the `camera` selector applies defaults to the `camera` object in `shotSetup`.  Is this correct?  If so maybe rename the `camera` selector to `cameraWithDefaults`?

Reply:
@shubhi1092 pointed this out in another [commit, and I gave this rationale](https://github.com/codecombat/codecombat/pull/5282#discussion_r281912984).

However I agree this is confusing as I broke this rule in the file. This has been turned into a cleanup task.